apps/api/src/modules/auth/auth.service.ts

```diff
@@ -41,6 +55,26 @@ export class AuthService {
       })
     } catch (e) {
       console.log(e)
+      throw new InternalServerErrorException()
     }
   }
+
+  async verfyPassword(loginDto: LoginDto, res: Response) {
+    const { password, email } = loginDto
+    const user = await this.repository.getUserByEmail(email)
+    if (!user) throw new UnauthorizedException('Wrong email or password')
+    if (!(await compare(password, user.password)))
+      throw new UnauthorizedException('Wrong email or password')
+    return this.createJwtToken(user, res)
+  }
+
+  async createJwtToken(user: User, res: Response) {
+    const { email, type } = user
+    const token: string = this.jwtService.sign({ email, type })
+    res.cookie('token', token, {
+      httpOnly: true,
+      secure: this.configService.get<boolean>('cookie.secure'),
```

Comment:
(Not sure) This field is not set in the configuration file. Where do you set `cookie.secure` config ?

Reply:
Yes not fill yet